tests/python/train/test_bucketing.py

```diff
@@ -101,9 +108,14 @@ def sym_gen(seq_len):
         num_epoch=num_epochs,
         batch_end_callback=mx.callback.Speedometer(batch_size, 50))
     logging.info('Finished fit...')
+    return model
+
+
+def test_bucket_module():
     # This test forecasts random sequence of words to check bucketing.
     # We cannot guarantee the accuracy of such an impossible task, and comments out the following line.
```

Comment:
not related to your PR. But what is the purpose of this line? Can it be made clearer?
Is it just a Note? then "comments out the following line" is not even needed!

Reply:
I think this comment is for 117 line which is commented.